utils/exporters/blender/addons/io_three/exporter/api/material.py

```diff
@@ -128,7 +128,7 @@ def normal_map(material):
 @_material
 def opacity(material):
     logger.debug('material.opacity(%s)', material)
-    return round(material.alpha - 1.0, 2);
+    return round( material.alpha, 2 );
```

Comment:
to make sure, I tried both cases.

I made a material with transparency-Alpha 0.3 in blender.
![screen shot 2015-01-21 at 6 30 36 am](https://cloud.githubusercontent.com/assets/212837/5826704/273931cc-a138-11e4-96f2-619908417e58.png)

then published with 3 type of exporters.

in exporter r69, transparency will be `"transparency": 0.30000001192092896`
```
				{
					"DbgColor": 15597568,
					"DbgIndex": 1,
					"DbgName": "red",
					"blending": "NormalBlending",
					"colorAmbient": [0.800000011920929, 0.0, 0.0],
					"colorDiffuse": [0.800000011920929, 0.0, 0.0],
					"colorEmissive": [0.0, 0.0, 0.0],
					"colorSpecular": [0.0, 0.0, 0.0],
					"depthTest": true,
					"depthWrite": true,
					"shading": "Lambert",
					"specularCoef": 50,
					"transparency": 0.30000001192092896,
					"transparent": true,
					"vertexColors": false
				}
```

in r70, opacity will be `"opacity": -0.7`
```
    },{
        "shading": "lambert",
        "colorAmbient": [0.800000011920929,0.0,0.0],
        "DbgIndex": 1,
        "depthWrite": true,
        "wireframe": false,
        "depthTest": true,
        "opacity": -0.7,
        "DbgName": "red",
        "visible": true,
        "colorEmissive": [0.0,0.0,0.0],
        "blending": "NormalBlending",
        "transparent": true,
        "doubleSided": true,
        "DbgColor": 15597568,
        "colorDiffuse": [0.800000011920929,0.0,0.0]
    }],
```

exporter of this pull request, opacity will be `"opacity": 0.3`
```
    },{
        "colorAmbient": [0.800000011920929,0.0,0.0],
        "transparent": true,
        "shading": "lambert",
        "DbgColor": 15597568,
        "colorDiffuse": [0.800000011920929,0.0,0.0],
        "blending": "NormalBlending",
        "depthTest": true,
        "visible": true,
        "depthWrite": true,
        "doubleSided": true,
        "colorEmissive": [0.0,0.0,0.0],
        "DbgName": "red",
        "DbgIndex": 1,
        "opacity": 0.3,
        "wireframe": false
    }],
```